app/views/idv/doc_auth/link_sent.html.erb

```diff
@@ -11,22 +11,26 @@
         message: flow_session[:error_message],
       ) %>
 <% end %>
-<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.text_message')) %>
 
+<%= render AlertComponent.new(type: :warning, class: 'margin-bottom-4') do %>
+  <strong class="display-block"><%= t('doc_auth.info.keep_window_open') %></strong>
+  <% if FeatureManagement.doc_capture_polling_enabled? %>
+    <%= t('doc_auth.info.link_sent_complete_polling') %>
+  <% else %>
+    <%= t('doc_auth.info.link_sent_complete_no_polling') %>
+  <% end %>
+<% end %>
+<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.text_message')) %>
 <div class="grid-row">
   <div class="grid-col-12 tablet:grid-col-3">
-    <%= image_tag asset_url('idv/phone.png'), width: 80, height: 119, alt: t('image_description.camera_mobile_phone') %>
```

Comment:
The `phone.png` file appears to be unused after this, can we remove it?

Reply:
@Kamal-Munshi ?

Reply:
@kbighorse - If this file isn't used, I'm +1 to removing it!

Reply:
deleted it